Dockerfile.tests

```diff
@@ -13,8 +13,8 @@ RUN apk --update add make openssl python \
 #  - v8.0.0027 (Vim 8, real async support)
 #  - v8.0.8069 (Updated Vim 8, https://groups.google.com/forum/#!topic/vim_dev/ro_FpeXgc24)
 RUN install_vim \
-        -tag v7.3.429 -name vim73 -build \
-        -tag v7.4.052 -name vim74-trusty -build \
+        -tag v7.3.429 -py -name vim73 -build \
+        -tag v7.4.052 -py -name vim74-trusty -build \
```

Comment:
https://github.com/neomake/neomake/pull/942#issuecomment-273927592 and the next.